### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**https://github.com/fivetran/dbt_shopify_source/issues/74

**This PR will result in the following new package version:** v0.10.0

Will require a full refresh to update the logic coming from dbt-date and dbt-expectations

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

Updates the dependency versions to the most current 

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

Run with a full refresh

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ n/a] DECISIONLOG updates have been updated (if applicable)
- [ ] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# dbt_shopify_source v0.10.0
+- Removes the dependency on [dbt-expectations](https://github.com/calogica/dbt-expectations/releases) and updates the [dbt-date](https://github.com/calogica/dbt-date/releases) dependency to the latest version. ([PR #75](https://github.com/fivetran/dbt_shopify_source/pull/75))
```
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: Can we add a sub header to this entry

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Small request to add more detail to the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-reneeli) [reply CHANGELOG.md:2]: updated

**Review by fivetran-joemarkiewicz — APPROVED** [review]
lgtm

**Pull request merged**